### PR TITLE
added a methods jest test

### PR DESCRIPTION
**Pull request by Ledoux**
@pocketjoso @franblas 
here are some jest test to test the utility methods of Teleport.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,17 +15,19 @@
...
   },
-  "engines": { "node": ">= 6.0" },
   "main": "lib/index.js",
   "name": "teleport.js",
   "scripts": {
     "compile": "babel --presets latest -d lib/ src/",
     "postinstall": "rm -rf lib && mkdir -p lib && npm run compile",
     "link": "source bin/link.sh",
-    "test": "jest",
+    "test": "export TEST_RANDOM=$RANDOM && jest tests/create.test.js && jest tests/utility_methods.test.js && jest tests/deploy_methods.test.js",
```
Comment (pocketjoso) [inline package.json]: jest by default picks up `test.js` files, so you don't need to specify them like this.

**Comment by pocketjoso** [comment]
I know this is work in progress, and the progress looks good. Some of the tests only log stuff and don't actually check the outcome of your method calls - if they don't they're not really useful. Guessing you're still working on this?

**Comment by Ledoux** [comment]
@pocketjoso 
tx, yes for the deploy test, I would need for example to make the test pinging the url of the deployed apps and expect a value from the ping.

File: package.json [hunk 2]
```diff
@@ -14,21 +15,23 @@
...
     "watch": "nodemon lib/index.js --watch src --exec \"npm run compile\""
   },
   "bin": {
     "tpt": "bin/index.js"
   },
-  "version": "0.1.1"
+  "version": "0.1.3"
```
Comment (franblas) [inline package.json]: Why not `0.1.2` ??

File: src/methods/install.js [hunk 3]
```diff
@@ -59,22 +59,52 @@ export function installScript () {
...
+  }
 }
 
 export function getInstallKubernetesCommand () {
   this.checkProject()
   const { kubernetes, project: { dir } } = this
   if (typeof kubernetes === 'undefined') {
-    this.consoleError('You must define a kubernetes config')
+    this.consoleWarn('You must define a kubernetes config')
     process.exit(1)
```
Comment (franblas) [inline src/methods/install.js]: If you exist the process it's better to write consoleError. Defining a kubernetes config is a requirement in this situation. @Ledoux

File: tests/test.config.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      .find(dir => re.test(dir))
+  if (testDir) {
+    TEST_RANDOM = testDir.split(re)[1]
+  }
+}
+const TEST_APP_NAME = `${TEST_PREFIX_APP_NAME}-${TEST_RANDOM}`
+const TEST_TEMPLATES = [
+  'teleport-express-webrouter',
+  'teleport-flask-websocket',
+  'teleport-heroku'
```
Comment (franblas) [inline tests/test.config.js]: Is it working if we add a frontend template to the list ?

**Review by franblas — COMMENTED** [review]
@Ledoux Greeeaat ! 👍 Made some comments

**Review by franblas — APPROVED** [review]
Let's merge it @Ledoux 🏆 Thaaaanks !!!

**Comment by pocketjoso** [comment]
@Ledoux @franblas 
screen shot 2017-02-03 at 12 36 53

**Comment by pocketjoso** [comment]
hopefully not related to Teleport - debugging myself.

**Comment by pocketjoso** [comment]
Seems to indeed by related by teleport - see slack communication.

**Pull request merged**